stack.yaml

```diff
@@ -1,22 +1,18 @@
-resolver: lts-13.26
+resolver: lts-17.6
 flags:
   aeson-pretty:
     lib-only: true
   these:
     assoc: false
-    quickcheck: false
 packages:
 - '.'
 extra-deps:
-# purescript 0.14.0-rc5
+# purescript 0.14.1
 - github: purescript/purescript
-  commit: 7ecc42669c69682996f2196ba2eef6c4ca827348
+  commit: 0d570f6e3828fc1a93c7c20252bccb04b7d2b0f6
```

Comment:
Can we switch back to using a released version of `purescript` now please?

Reply:
Yeah. One downside of this is that `purescript` depends on `purescript-ast`, which is going to be merged into `purescript-cst`. So, we'll need to update this again to drop `purescript-ast` once `v0.14.2` is released.

Reply:
That's fine, we'll need to update this file at that point anyway.